build.gradle

```diff
@@ -1062,5 +1062,3 @@ task setup() {
     dependsOn ':clean:extractMapped'
     dependsOn ':forge:extractMapped' //These must be strings so that we can do lazy resolution. Else we need evaluationDependsOnChildren above
 }
-
-
```

Comment:
Is this meant to be part of the diff? As it is the only changes in the file.

Reply:
It wasn't intended to be but idk why there's a double newline at the end of it